### DISCUSSION TITLE
Base16 themes updated

### DISCUSSION THREAD

**Discussion by rgieseke** [orbitalquark/textadept#476] [Show and tell]
The base16 based themes have been updated for Textadept 12: https://github.com/rgieseke/base16-textadept

I used [Tinted Theming](https://github.com/tinted-theming).

I haven't checked everything, so let me know if some styling can be improved.

**Comment by georgeraraujo** [comment]
I didn't know about these themes, I just tried some and they look GREAT! Thanks for sharing!
